.github/workflows/language_server_simulator_nballerina.yml

```diff
@@ -0,0 +1,83 @@
+name: Language Server Simulator on nBallerina
+
+on:
+  schedule:
+    - cron: '0 */12 * * *'
+  workflow_dispatch:
+
+jobs:
+  run_simulator:
+    name: Run LS Simulator
+    runs-on: ubuntu-latest
+    timeout-minutes: 240
+    strategy:
+      fail-fast: false
+      matrix:
+        branch: [ "master", "2201.8.x", "2201.7.x" ]
+        skipGenerators: [ "" ]
```

Comment:
Why do we need this?

Reply:
This is required to skip a particular generator and run the simulator. This functionality is needed when we are trying to narrow down a cause for a memory leak.

Reply:
I mean adding `""` empty string in matrix isn't needed at all. You can simply pass it the step. If you use more than one string for `skipGenerators`, you should use matrix.

Reply:
Added another case to skip